src/pages/list/table-list/index.tsx

```diff
@@ -0,0 +1,490 @@
+
+const FormItem = Form.Item;
+const { Option } = Select;
+const getValue = (obj: { [x: string]: string[] }) =>
+  Object.keys(obj)
+    .map(key => obj[key])
+    .join(',');
+
+type IStatusMapType = 'default' | 'processing' | 'success' | 'error';
+const statusMap = ['default', 'processing', 'success', 'error'];
+const status = ['关闭', '运行中', '已上线', '异常'];
+
+interface TableListProps extends FormComponentProps {
+  dispatch: Dispatch<
+    Action<
+      | 'listAndtableList/add'
+      | 'listAndtableList/fetch'
+      | 'listAndtableList/remove'
+      | 'listAndtableList/update'
+    >
+  >;
+  loading: boolean;
+  listAndtableList: StateType;
+}
+
+interface TableListState {
+  modalVisible: boolean;
+  updateModalVisible: boolean;
+  expandForm: boolean;
+  selectedRows: TableListItem[];
+  formValues: { [key: string]: string };
+  stepFormValues: Partial<TableListItem>;
+}
+
+/* eslint react/no-multi-comp:0 */
+@connect(
+  ({
+    listAndtableList,
+    loading,
+  }: {
+    listAndtableList: StateType;
+    loading: {
+      models: {
+        [key: string]: boolean;
+      };
+    };
+  }) => ({
+    listAndtableList,
+    loading: loading.models.listAndtableList,
+  }),
+)
+class TableList extends Component<TableListProps, TableListState> {
+  state: TableListState = {
+    modalVisible: false,
+    updateModalVisible: false,
+    expandForm: false,
+    selectedRows: [],
+    formValues: {},
+    stepFormValues: {},
+  };
+
+  columns: StandardTableColumnProps[] = [
+    {
+      title: '规则名称',
+      dataIndex: 'name',
+    },
+    {
+      title: '描述',
+      dataIndex: 'desc',
+    },
+    {
+      title: '服务调用次数',
+      dataIndex: 'callNo',
+      sorter: true,
+      align: 'right',
+      render: (val: string) => `${val} 万`,
+      // mark to display a total number
+      needTotal: true,
+    },
+    {
+      title: '状态',
+      dataIndex: 'status',
+      filters: [
+        {
+          text: status[0],
+          value: '0',
+        },
+        {
+          text: status[1],
+          value: '1',
+        },
+        {
+          text: status[2],
+          value: '2',
+        },
+        {
+          text: status[3],
+          value: '3',
+        },
+      ],
+      render(val: IStatusMapType) {
+        return <Badge status={statusMap[val]} text={status[val]} />;
+      },
+    },
+    {
+      title: '上次调度时间',
+      dataIndex: 'updatedAt',
+      sorter: true,
+      render: (val: string) => <span>{moment(val).format('YYYY-MM-DD HH:mm:ss')}</span>,
+    },
+    {
+      title: '操作',
+      render: (text, record) => (
+        <Fragment>
+          <a onClick={() => this.handleUpdateModalVisible(true, record)}>配置</a>
+          <Divider type="vertical" />
+          <a href="">订阅警报</a>
+        </Fragment>
+      ),
+    },
+  ];
+
+  componentDidMount() {
+    const { dispatch } = this.props;
+    dispatch({
+      type: 'listAndtableList/fetch',
+    });
+  }
+
+  handleStandardTableChange = (
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'handleStandardTableChange' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)